aws/v15.1.1/release.diff

```diff
@@ -0,0 +1,71 @@
+    version: 2.2.0                                              |      version: 2.2.1
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.7.1                                                     version: 1.7.1
+  - componentVersion: 1.3.1                                          - componentVersion: 1.3.1
+    name: cert-manager                                                 name: cert-manager
+    version: 2.8.0                                                     version: 2.8.0
+  - name: chart-operator                                             - name: chart-operator
+    version: 2.18.0                                                    version: 2.18.0
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.20.3                                                    version: 1.20.3
+  - componentVersion: 1.8.3                                          - componentVersion: 1.8.3
+    name: coredns                                                      name: coredns
+    version: 1.6.0                                                     version: 1.6.0
+  - componentVersion: 0.8.0                                          - componentVersion: 0.8.0
+    name: external-dns                                                 name: external-dns
+    version: 2.4.0                                                     version: 2.4.0
+  - componentVersion: 4.1.0                                          - componentVersion: 4.1.0
+    name: kiam                                                         name: kiam
+    version: 2.0.0                                                     version: 2.0.0
+  - componentVersion: 1.9.7                                          - componentVersion: 1.9.7
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.3.1                                                     version: 1.3.1
+  - componentVersion: 0.4.1                                          - componentVersion: 0.4.1
+    name: metrics-server                                               name: metrics-server
+    version: 1.3.0                                                     version: 1.3.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.10.2                                                    version: 1.10.2
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.7.2                                                     version: 1.7.2
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    releaseOperatorDeploy: false                                <
```

Comment:
`releaseOperatorDeploy`: false was removed, but the line is present for v15.1.0. Is false the default value?

Reply:
thanks, fixed 👍🏻